src/replicate_stage.rs

```diff
@@ -328,7 +329,7 @@ mod test {
         let exit = Arc::new(AtomicBool::new(false));
         let (replicate_stage, ledger_writer_recv) = ReplicateStage::new(
             Arc::new(my_keypair),
-            Arc::new(vote_account_keypair),
+            Arc::new(vote_account_id),
```

Comment:
pubkeys can be passed by value or by reference, they're simple

Reply:
👍

Reply:
except that we should always pass them by reference unless the callee is going to store them in a struct. Historically, we've all been super inconsistent about that because it implements Copy, so you can't see when you're accidentally cloning 32-byte arrays.

Reply:
The vote_account_id is being passed to thread replicate_thread. I am getting error if I pass it as a reference. I see passing as value has it's overheads (copying of 32 bytes). Arc seems to be right thing to use. Is that correct?